app/main.js

```diff
@@ -1,163 +1,229 @@
-const { app, BrowserWindow, ipcMain, shell } = require('electron');
+const electron = require('electron');
+const {
+    app,
+    BrowserWindow,
+    ipcMain, 
+    Menu, 
+    shell
+} = electron;
 const fs = require('fs-extra');
 const storage = require('electron-json-storage');
 const _ = require('lodash');
 const SWProxy = require('./proxy/SWProxy');
-
 const path = require('path');
 const url = require('url');
-
 global.gMapping = require('./mapping');
-
 global.win;
 let defaultFilePath = path.join(app.getPath('desktop'), app.getName() + ' Files');
+let template = [{
+  label: 'Window',
+  role: 'window',
+  submenu: [{
```

Comment:
See below.
Also, I didn't initially implement maximize because considering the relatively compact interface you guys have designed, and the very hands-off/background way in which the app works (as opposed to something like Photoshop let's say that is very hands on), I couldn't really see a reason you would need it fullscreen.